CHANGELOG.md

```diff
@@ -14,6 +14,15 @@ All notable changes to this project will be documented in this file, in reverse
   factory `Zend\Di\Container\GeneratorFactory` for creating a
   `Zend\Di\CodeGenerator\InjectorGenerator` instance with zend-servicemanager.
 
+- [#38](https://github.com/zendframework/zend-di/pull/38) adds `Zend\Di\Resolver\InjectionInterface`
+  to define the contract 
```

Comment:
Define _which_ contract? Elaborate here, as this is where users get the information they need to determine the impact of the changes.

Reply:
Right, that's related to the discussion above. This will need a change indeed.

Reply:
Besides this little bit, this is ready to :ship: